README.md

```diff
@@ -512,16 +512,17 @@ Pass any custom view in story view. It will be rendered on top of story view as
+> | Name                |           Default            | Type                                   | <div style="width:290px">Description</div>                                                                                                                           |
+> | :------------------ | :--------------------------: | :------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+> | **stories\***       |          undefined           | StoriesType[]                          | Array of multiple user stories                                                                                                                                       |
+> | **visible\***       |            false             | boolean                                | Hide / show story view                                                                                                                                               |
+> | userStoryIndex      |              0               | number                                 | Pass clicked index of horizontal user list.                                                                                                                          |
+> | storyContainerProps |              {}              | StoryContainerProps                    | Customize all story props, detailed props in below `StoryContainer` section                                                                                          |
+> | onChangePosition    |             null             | (progressIndex, userIndex) => {}       | Callback when progress index changes                                                                                                                                 |
+> | transitionMode      |     TransitionMode.Cube      | TransitionMode: {Default, Cube, Scale} | To customize user story transition, (TransitionMode.default : no transition, Transition.scale : zoomIn/zoomOut transition, Transition.Cube: 3D cube transition) cube |
+> | onComplete          |             null             | () => {}                               | Callback when stories closed or complete                                                                                                                             |
+> | `props`             |              -               | StoryContainerProps                    | Pass any `StoryContainerProps` props to customize story                                                                                                              |
+> | pointers            | { pageX: 0,<br /> pageY: 0 } | PointerType                            | This will use for modal opening and closing position. you can pass here the position of profile avatar's positions offsets.                                          |
```

Comment:
@prince-d-simform We should revise this description as follows
e.g Pointers will be used internally for modal opening and closing positions, You can pass profile avatar's position offsets here.